### ISSUE TITLE
RCNN with Resnet in the future?

### ISSUE THREAD

**Issue by mortengryning** [Microsoft/CNTK#1171]
I have used the Alexnet R-CNN example to train my own neural network. And the results are pretty good. But when reading up on AlexNet vs Resnet, Resnet performs much better. I would like that extra edge.

Are there any plans to combine Resnet with the fast R-CNN example? I think that would be a very valuable contribution to the CNTK project, as it would provide us (the users) with state-of-the-art image recognition out of the box.

Also, compared to Caffe, I really like that images are stored as images in the datasets. This makes it more intuitive to use CNTK compareds to other framework, and the files for training were also a great addition :)

**Comment by cha-zhang** [comment]
Yes, we have plans for ResNet R-CNN. Currently we are retraining ResNet models to match the Facebook implementation numbers. Once that is done, we will do ResNet R-CNN.

**Comment by mortengryning** [comment]
Great news :-). I will look forward to the implementation. The Alexnet is already so much better than the "old" methods so I can't wait to test the resnet.

**Comment by mahilleb-msft** [comment]
@pkranen, @cha-zhang - do you know the state of this?
